docs/contribute/getting-started.md

```diff
@@ -30,301 +30,150 @@ page_nav:
-<li><a href="https://github.com/liabru/jquery-match-height">jquery-match-height</a> (<a href="https://github.com/liabru/jquery-match-height/blob/master/LICENSE">MIT License</a>)</li>
-<li><a href="https://github.com/simple-icons/simple-icons">simple-icons</a> (<a href="https://github.com/simple-icons/simple-icons/blob/develop/LICENSE.md">CC0-1.0 License</a>)</li>
-<li><a href="https://github.com/colebemis/feather">feather</a> (<a href="https://github.com/colebemis/feather/blob/master/LICENSE">MIT License</a>)</li>
-<li><a href="https://fonts.google.com/specimen/Montserrat">Montserrat</a> (<a href="http://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&amp;id=OFL">SIL Open Font License, 1.1</a>)</li>
-<li><a href="https://fonts.google.com/specimen/Noto+Sans">Noto Sans</a> (<a href="http://www.apache.org/licenses/LICENSE-2.0.html">Apache License, version 2.0</a>)</li>
-<li><a href="https://fonts.google.com/specimen/Source+Code+Pro">Source Code Pro</a> (<a href="http://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&amp;id=OFL">SIL Open Font License, 1.1</a>)</li>
-</ul>
-
-                </div><!-- /.content -->
-            </div><!-- /.col -->
-        </div><!-- /.row -->
-</div><!-- /.container -->
+## What is the SingularityNET Developer Portal Theme?
+
+The SingularityNET Developer Portal Theme is [Jekyll](https://jekyllrb.com/) theme created for project documentations.  
+You can use it with [GitHub](https://pages.github.com/) and [GitLab Pages](https://about.gitlab.com/features/pages/) as well as a standalone project.
+
+## What is Jekyll?
+
+[Jekyll](https://jekyllrb.com/) is a simple, blog-aware, static site generator. It takes a template directory containing raw text files in various formats, runs it through a converter (like [Markdown](https://daringfireball.net/projects/markdown/)) and [Liquid](https://github.com/Shopify/liquid/wiki) renderer, and spits out a complete, ready-to-publish static website suitable for serving with your favourite web server.
+
+>__Full Jekyll documentation__ <br> You can find full Jekyll documentation [here](https://jekyllrb.com/docs/home/).
+
+## Install Jekyll
+
+### Requirements
+
+Installing Jekyll should be straight-forward if all requirements are met. Before you start, make sure your system has the following:
+
+*   GNU/Linux, Unix, or macOS
+*   [Ruby](https://www.ruby-lang.org/en/downloads/) version 2.0 or above, including all development headers
+*   [RubyGems](https://rubygems.org/pages/download)
+*   [GCC](https://gcc.gnu.org/install/) and [Make](https://www.gnu.org/software/make/) (in case your system doesn’t have them installed, which you can check by running `gcc -v` and `make -v` in your system’s command line interface)
+
+### Install with RubyGems
+
+We suggest installing Jekyll via [RubyGems](https://rubygems.org/pages/download). On a terminal prompt, run the following commands:
+
+```
+$ gem install jekyll
+$ gem install jekyll-redirect-from
```

Comment:
I have since added `jekyll-paginate`, `jekyll-sitemap`, and `jemoji` which also should be installed.

I have also added details to the repo README.md as this is usually where people will look to get started with a project. https://github.com/singnet/dev-portal#setup

I think we should maybe point people to the README for setup of requirements, and the README probably should have some the details here added to it. This should hopefully avoid us having duplicate content that becomes out of date.

Reply:
Removed “Start development server” section and added a link to the Setup section of this repo.